APPENGINE.md

```diff
@@ -0,0 +1,31 @@
+Use Google Cloud Java Client with GoogleAppEngine
```

Comment:
The official name has spaces ("Google App Engine") - change it to this in all locations.